.golangci.yml

```diff
@@ -62,6 +65,22 @@ linters-settings:
       # linters that related to local tool, so they should be separated
       - localmodule
 
+  depguard:
+    rules:
+      # enforce the library has no dependencies except the standard library
+      code:
+        files:
+          - "!$test"  # depguard alias for all files except the test ones
```

Comment:
That's nice, so it says that you can't import testing library in non-test files? I like it.

Reply:
This exact line says for all files that are not tests files, apply the following rule

and the rule is about having nothing but standard library import